scripts/index.js

```diff
@@ -0,0 +1,154 @@
+        lanscapeContent.textContent = `${weather_info.freezing.emojis}`;
+    }
+}
+
+const changeTempValue = () => {
+    const tempValue = document.querySelector("#tempValue");
+    tempValue.textContent = `Temperature: ${state.temperature}F°`;
+    const tempDisplay = document.querySelector("#tempDisplay");
+    tempDisplay.textContent = `${state.temperature}F°`;
+}
+
+const decreaseTemp = (event) => {
+    state.temperature -= 1;
+    changeTempValue()
+    changeTempColor()
+    changeLandscape()
+};
+
+const increaseTemp = (event) => {
+    state.temperature += 1;
+    changeTempValue()
+    changeTempColor()
+    changeLandscape()
+};
+
+const changeSky = (event) => {
+    const skyContent = document.querySelector("#skyContent");
+    const mainBackground = document.querySelector("#mainBackground");
+    const mainContent = document.querySelector(".mainContent");
+
+    if(event.target.options.selectedIndex == 0){
+        mainContent.style.backgroundColor = "#c9c9c3"
+        mainBackground.style.backgroundImage = `url(${sky_info.sunny})`
+        skyContent.textContent = `☁️ ☁️ ☁️ ☀️ ☁️ ☁️`;
+    }else if(event.target.options.selectedIndex == 1){
+        mainContent.style.backgroundColor = "#cadeed"
+        mainBackground.style.backgroundImage = `url(${sky_info.cloudy})`
+        skyContent.textContent = `☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️`;
+    }else if(event.target.options.selectedIndex == 2){
+        mainContent.style.backgroundColor = "#b7f6f7"
+        mainBackground.style.backgroundImage = `url(${sky_info.rainy})`
+        skyContent.textContent = `🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧`;
+    }else if(event.target.options.selectedIndex == 3){
+        mainContent.style.backgroundColor = "#e3e2de"
+        mainBackground.style.backgroundImage = `url(${sky_info.snowy})`
+        skyContent.textContent = `🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨`;
+    }
+}
+
+const changeCityName = (event) => {
+    state.cityName = event.target.value
+    const cityName = document.querySelector("#cityName");
+    cityName.textContent = state.cityName;
+}
+
+const clearCityName = () => {
+    state.cityName = "Antioch, CA"
+    const cityName = document.querySelector("#cityName");
+    cityName.textContent = state.cityName;
+    const cityNameSelect = document.querySelector("#cityNameSelect");
+    cityNameSelect.value = ""
+}
+
+const registerEventHandlers = (event) => {
+    const tempDownButton = document.querySelector("#tempDown");
+    tempDownButton.addEventListener("click", decreaseTemp);
+    const tempUpButton = document.querySelector("#tempUp");
+    tempUpButton.addEventListener("click", increaseTemp);
+    const skySelect = document.querySelector("#sky");
+    skySelect.addEventListener("change", changeSky);
+    const renameCity = document.querySelector("#cityNameSelect");
+    renameCity.addEventListener("input", changeCityName);
+    const resetCityName = document.querySelector("#resetButton");
+    resetCityName.addEventListener("click", clearCityName);
+
```

Comment:
Similar to the suggestion to call the functions to update the temperature UI from here, also call the sky and city updates, so that everything reflects the default data values.